versions/wcoss2.ver

```diff
@@ -3,7 +3,8 @@ export prod_envir_ver=${prod_envir_ver:-2.0.4} # Allow override from ops ecflow
 export prod_util_ver=${prod_util_ver:-2.0.9}   # Allow override from ops ecflow
 
 export obsproc_run_ver=1.3.0
-export prepobs_run_ver=1.2.0
+export prepobs_run_ver=1.1.0
```

Comment:
Remove the change to `prepobs_run_ver`, it should still be `export prepobs_run_ver=1.2.0`. I'll actually be merging in an update to make it 1.3.0 after GFSv16.3.20 goes into ops in early December. I'll take care of that update when it happens.

Reply:
Fixed